README.md

```diff
@@ -40,9 +40,12 @@ The downsides of using YouTube IFrame Player API are:
 /**
  * A factory function used to produce an instance of YT.Player and queue function calls and proxy events of the resulting object.
  *
- * @param {HTMLElement|String} elementId Either An existing YT.Player instance,
+ * @param {Object|HTMLElement|String} elementId Either An existing YT.Player instance,
```

Comment:
`Object` is obscure (`HTMLElement` is an object too). Please define a custom type using [`@typedef`](http://usejsdoc.org/tags-typedef.html) and describe it as `YT.Player` with a link to the YT.Player docs.